bigtable/tests/unit/test_client.py

```diff
@@ -93,13 +93,16 @@ def test_credentials_getter(self):
         project = 'PROJECT'
         client = self._make_one(
             project=project, credentials=credentials)
-        self.assertIs(client._credentials, credentials)
+        self.assertIs(client._credentials,
+                      credentials.with_scopes.return_value)
 
     def test_project_name_property(self):
         credentials = _make_credentials()
+        channel = _make_channel()
         project = 'PROJECT'
         client = self._make_one(
-            project=project, credentials=credentials, admin=True)
+            project=project, channel=channel, credentials=credentials,
+            admin=True)
```

Comment:
If we are going to pass `channel` here, shouldn't there be an assertion that it was saved correctly?